mlflow/langchain/__init__.py

```diff
@@ -85,6 +88,10 @@
 FLAVOR_NAME = "langchain"
 _MODEL_TYPE_KEY = "model_type"
 
+# The following constants is used in flavor configuration to specify the code paths
+# that should be set using _set_code_path when the model is loaded by code.
+_CODE_CONFIG = "_code_config"
```

Comment:
Can we move this to `_rag_utils`  update the comment to indicate that it's RAG-specific?